src/uikit.cs

```diff
@@ -18285,6 +18292,10 @@ interface UIDocumentBrowserViewControllerDelegate {
 
 		[Export ("documentBrowser:willPresentActivityViewController:")]
 		void WillPresent (UIDocumentBrowserViewController controller, UIActivityViewController activityViewController);
+
+		[iOS (12,0)]
+		[Export ("documentBrowser:didPickDocumentsAtURLs:")]
+		void DidPickDocumentAtUrls (UIDocumentBrowserViewController controller, NSUrl[] documentUrls);
```

Comment:
`DidPickDocumentAtUrls` -> `DidPickDocument`

Reply:
And it's `DidPickDocuments` with an `s` (:

Reply:
@dalexsoto we talked about this online, DidPickDocumentUrls was deprecated over this new method, which is a rename, but is exactly the same one. Since it is a protocol, I have to create the new method, and the Urls is needed in the name, since is not a document what is pick, but the urls. I'll address the 's' from vincent. Will leave Urls.